installer/genesis/devnet/genesis.json

```diff
@@ -17,276 +17,249 @@
         "voteLst": 3000000
       }
     },
     {
-      "addr": "QBQ5ODCH6PWCB5TFQVCOFLHSJKN6NYYB3E3HPCJJTTNRVM5XD3B3KPWFJA",
+      "addr": "D5HH5IRCH2TPBGY5G7CYHFBY6WBQLWKYFUCG2LTKPJLJQV2T5X2AF2FGTU",
       "comment": "Wallet7",
       "state": {
         "algo": 500000000000000,
         "onl": 1,
-        "sel": "cxhpl3wBnc9bVW6G5ePiya8nzPvUUaprIoTWvTXJmpY=",
-        "vote": "W8tVBEdWGDArQFj43qXDw18LRJKaGjgZWUv54tsus+A=",
+        "sel": "CEfy71Id8IpNAEwS/V3SXmrBBQktdmKekD8DVNvKVe4=",
+        "vote": "yrFaO6g1Ln9XvOT9KoccW06n9vpf9BbjTMtJWN13cls=",
         "voteKD": 10000,
         "voteLst": 3000000
       }
     },
     {
-      "addr": "WILFZPBQMJ7GYSPNA6JPXIZZUZ27XWDJ2NKC7WIODXXGOF73C4TEDEHE3U",
+      "addr": "WB25K2CNUZHJZDZC65JSDZK4DCB7PJS2HY6CENEDFIWDAGY5PUMESWLQNA",
       "comment": "Wallet8",
       "state": {
         "algo": 500000000000000,
         "onl": 1,
-        "sel": "SMSZnIm3GSD1Dr5AEq5ksxxCIqPIrn19WSDNtJSJuiM=",
-        "vote": "absDJtKafyc4KbrB1GnLcFGv8xQjuqB46gPXXTpIsTU=",
+        "sel": "faWUNU08900giL3F52zDgPSo075mGzJbFwfJLSVRdbY=",
+        "vote": "DMuzD8oVB6Qb7BEoiWD6iIwmRX1jS8OTFVu8b6aSL08=",
         "voteKD": 10000,
         "voteLst": 3000000
       }
     },
     {
-      "addr": "4LSVIF5EY2BQZ6GENL6GT4SQTKX3UVDTBDJXFESGY7GI7FXHZA522KMG5I",
+      "addr": "D2SP4JPC37SXIER4G4CQUQV4SYTSSXYI4UF2GFUZ7YWZUMC2MOGTQLKNEI",
       "comment": "Wallet9",
       "state": {
         "algo": 500000000000000,
         "onl": 1,
-        "sel": "h3aKVPyQSpXAI2zcDD2f1CCBqf9fQd4Z+oJjQ6erFLI=",
-        "vote": "nNfIrFtpPR6zo4hH6F7HCMA44PDcNOEWtDVRf7dnKe4=",
-        "voteKD": 10000,
-        "voteLst": 3000000
-      }
-    },
-    {
-      "addr": "IDUTJEUIEVSMXTU4LGTJWZ2UE2E6TIODUKU6UW3FU3UKIQQ77RLUBBBFLA",
-      "comment": "bank",
-      "state": {
-        "algo": 100000000000000,
-        "onl": 1,
-        "sel": "6Rax2zpOehfDXHyyvMFJwc22A//Pm6rxW4MiEBGIwog=",
-        "vote": "5JbOqqGn1Ypw+uH3t2i61DcjWYo1AslT0jyOGgcVC6Y=",
+        "sel": "skRYnkIpe3dnYe58Vi4QStzNeI3TnhOhmiQTdQsipGc=",
+        "vote": "vA4HN3yLMKrisDFkLSp5tZ3Ud6oUXs2eAJUamj1YfyI=",
         "voteKD": 10000,
         "voteLst": 3000000
       }
-    },
-    {
-      "addr": "PNWOET7LLOWMBMLE4KOCELCX6X3D3Q4H2Q4QJASYIEOF7YIPPQBG3YQ5YI",
-      "comment": "pp1",
-      "state": {
-        "algo": 50000000000000
-      }
-    },
-    {
-      "addr": "F5FALDSUFYO5LQU4OQY2HPZTXSUPRHUAERZ7IDG5QCBC76AOHEPK2VUJ34",
-      "comment": "pp2",
-      "state": {
-        "algo": 50000000000000
-      }
     }
   ],
   "fees": "A7NMWS3NT3IUDMLVO26ULGXGIIOUQ3ND2TXSER6EBGRZNOBOUIQXHIBGDE",
-  "id": "v1.0",
+  "id": "v1",
   "network": "devnet",
-  "proto": "https://github.com/algorand/spec/tree/a26ed78ed8f834e2b9ccb6eb7d3ee9f629a6e622",
-  "rwd": "7777777777777777777777777777777777777777777777777774MSJUVU",
-  "timestamp": 1560210489
+  "proto": "https://github.com/algorandfoundation/specs/tree/3a83c4c743f8b17adfd73944b4319c25722a6782",
```

Comment:
Also curious about starting this from most recent protocol, rather than first protocol. Though I suppose this doesn't matter because the mainnet network also did not start with the first.

Reply:
Tsachi asked to confirm that it is the latest, which it is.

Reply:
it is ConsensusV24, which is currently the current default one:
```
const ConsensusCurrentVersion = ConsensusV24
```